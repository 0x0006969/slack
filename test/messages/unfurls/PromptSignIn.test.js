const PromptSignIn = require('../../../lib/messages/unfurls/prompt-sign-in');

describe('Unfurl PromptSignIn message rendering', () => {
  test('works', async () => {
    const message = new PromptSignIn();
    expect(message.getAttachment()).toMatchSnapshot();
  });
});
