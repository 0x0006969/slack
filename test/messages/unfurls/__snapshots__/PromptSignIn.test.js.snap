// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unfurl PromptSignIn message rendering works 1`] = `
Object {
  "actions": Array [
    Object {
      "style": "primary",
      "text": "Connect GitHub account",
      "type": "button",
      "url": "https://some-url.com/to-log-in?state=probably-with-state",
    },
  ],
  "color": "#24292f",
  "mrkdwn_in": Array [
    "text",
  ],
  "text": "To see the rich preview, please connect your GitHub account. You will only have to do this once.",
}
`;
